src/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -59,6 +60,9 @@ internal ApplicationConfiguration()
         public AuthorityInfo DefaultAuthorityInfo => Authorities.Single(x => x.IsDefault);
         public string Component { get; internal set; }
 
+        internal ILegacyCachePersistence UserTokenLegacyCachePersistenceForTest { get; set; }
```

Comment:
what does LegacyCache refer to? In msal v2 it's the adal v3 cache.

Reply:
Correct.  That meaning hasn't changed.

---
In reply to: [247638823](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/785#discussion_r247638823) [](ancestors = 247638823)